api/config/cdr_config_prod.json

```diff
@@ -7,8 +7,17 @@
       "servicePerimeter": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_prod",
       "authDomainName": "all-of-us-registered-prod",
       "authDomainGroupEmail": "all-of-us-registered-prod@firecloud.org",
-      "datasetsBucket": "gs://fc-aou-datasets-registered/",
       "enableUserWorkflows": false
+    },
+    {
+      "accessTierId": 2,
+      "shortName": "controlled",
+      "displayName": "Controlled Tier",
+      "servicePerimeter": "terra_prod_aou_prod_2",
+      "authDomainName": "all-of-us-controlled-prod",
+      "authDomainGroupEmail": "all-of-us-controlled-stable@firecloud.org",
```

Comment:
typo: stable

Reply:
Great catch - that might have been very painful to track down afterwards.